Gruntfile.js

```diff
@@ -99,7 +99,11 @@ module.exports = function (grunt) {
     grunt.registerTask('compile:js', function (fullCompile) {
         grunt.task.run(['clean:js', 'compile:inlineSvgs']);
 
-        grunt.task.run(['concurrent:requireJS', 'copy:javascript']);
+        if (!options.isDev) {
```

Comment:
i'm not sure why? if you still compile the JS in `--dev`, you'll have to wait for it to compile but never download it. not sure what the benefit is? it was the same with JSPM, we never bundled in dev. same with uglifying etc?